src/controllers/auth.js

```diff
@@ -0,0 +1,169 @@
+import UserManager from '../components/userManager';
+import Utils from "../components/utils";
+import AuthManager from "../components/authManager"
+import Db from "../components/db"
+
+class AuthController {
+
+    async generateAuthJwt(req, res) {
+        const did = req.body.did;
+        const contextName = req.body.contextName;
+        const authJwt = AuthManager.generateAuthJwt(did, contextName)
+
+        return res.status(200).send({
+            status: "success",
+            authJwt
+        });
+    }
+    
+    /**
+     * Authenticate the user with a signed consent message to obtain a refresh token.
+     * 
+     * Will automatically create the user if they don't already exist.
+     * 
+     * Returns a refresh token, access token and database hostname for connections
+     * 
+     * @param {*} req 
+     * @param {*} res 
+     * @returns 
+     */
+    async authenticate(req, res) {
+        const authJwt = req.body.authJwt;
+        const did = req.body.did;
+        const contextName = req.body.contextName;
+        const signature = req.body.signature;
+        const deviceId = req.body.deviceId;
```

Comment:
I would just destructure at this many keys, makes it a bit more concise.

Reply:
Agreed